javascript/javascript2/week1/find.js

```diff
@@ -0,0 +1,33 @@
+const danishString = ["Blå grød med røde bær"];
+const str1 = danishString.toString();
+
+const danishString2 = ["Jeg har en blå bil"];
+const str2 = danishString2.toString();
+
+function alphabet(array) {
+  let letter1 = 0;
+  let letter2 = 0;
+  let letter3 = 0;
```

Comment:
Overall good work!
suggestion:
Give meaningful names to function and variables.
ex: function name can be danishAlphabetsCount( )